src/utils/leetcode_util.py

```diff
@@ -106,6 +106,108 @@ def query_builder_user_rank(leetcode_username: str):
         }
         return query_user_rank
 
+    @staticmethod
+    def query_builder_user_submissions(leetcode_username: str, skip: int = 0):
+        """
+        Builds a leetcode query to gather the provided user's submissions
+        """
+        query_user_submissions = {
+            "query": """
+                query userSolutionTopics($username: String!, $orderBy: TopicSortingOption, $skip: Int, $first: Int) {
+                    userSolutionTopics(
+                        username: $username
+                        orderBy: $orderBy
+                        skip: $skip
+                        first: $first
+                    ) {
+                        pageInfo {
+                            hasNextPage
+                        }
+                        edges {
+                            node {
+                                id
+                                title
+                                url
+                                viewCount
+                                questionTitle
+                                post {
+                                    creationDate
+                                    voteCount
+                                }
+                            }
+                        }
+                    }
+                }
+            """,
+            "variables": {
+                "username": leetcode_username,
+                "orderBy":"newest_to_oldest",
+                "skip":skip,
```

Comment:
`first` should be `skip` here, because it likely represents an offset into the answers